smart_card_connector_app/src/window.css

```diff
@@ -175,3 +175,34 @@ body {
   top: -2px;
   width: 20px;
 }
+
+.tool-tip {
```

Comment:
I see, so this is a hand-written tooltip. Can we use the standard one? I think it's the "title" attribute: https://www.w3schools.com/tags/att_title.asp

The benefit of the standard tooltip is that we can just rely on the browser's implementation of this feature without having to maintain our own implementation. Also the browser's standard implementation is more coherent with the typical UX on the target platform: i.e., how the tooltip is styled, when it appears (usually tooltips are only displayed after some timeout), etc.

Reply:
Alright that's better, thanks for the info